.github/workflows/nix.yaml

```
@@ -1,151 +1,72 @@
+    name: Checks (${{ matrix.check }})
+    if: needs['setup-checks-jobs'].outputs.continue == 'true'
+    runs-on: ubuntu-latest
+    needs: [setup-checks-jobs]
     permissions:
       contents: read
-      packages: write
     strategy:
       fail-fast: false
-      matrix:
-        include: ${{ fromJSON(needs['setup-packages-jobs'].outputs.matrix) }}
+      matrix.include: ${{ fromJSON(needs['setup-checks-jobs'].outputs.matrix) }}
```

actionlint (1.7.12)

[error] unexpected key "matrix.include" for "strategy" section. expected one of "fail-fast", "matrix", "max-parallel"

(syntax-check)